### DISCUSSION TITLE
How does the sketchybar-app-font work?

### DISCUSSION THREAD

**Discussion by juliettepaul** [FelixKratz/SketchyBar#279] [Q&A] [answered]
I've noticed that my bar at the top shows all these `:firefox:` and `:calendar:` sections. I tried to install the sketchybar-app-font but I'm not sure how to make it work. How is everyone else getting their icons to show up in the bar?

**Comment by FelixKratz** [comment, marked as answer]
You only need to install the font:
```bash
curl -L https://github.com/kvndrsslr/sketchybar-app-font/releases/download/v1.0.3/sketchybar-app-font.ttf -o $HOME/Library/Fonts/sketchybar-app-font.ttf
```
and everything should be working. Now you can see the font in Font Book and use its ligatures to display app icons.

**Reply by juliettepaul** [reply]
I have the file there now. Do I need to set this as my main font in the bar settings? It doesn't look like you do this. This is what I'm seeing: 
![image](https://user-images.githubusercontent.com/154509/199560333-18ba4a24-3720-43be-8025-b623268c212f.png)
I would expect to see icons there. Any tips

**Reply by juliettepaul** [reply]
Yeah I'm still not seeing it in Font book after that step. Adding it manually in font book just says validating and then nothing happens. Going to investigate that issue more.

**Reply by FelixKratz** [reply]
Sorry, thats my bad, the curl was bad. Try this:
```bash
curl -L https://github.com/kvndrsslr/sketchybar-app-font/releases/download/v1.0.3/sketchybar-app-font.ttf -o $HOME/Library/Fonts/sketchybar-app-font.ttf
```
should work now!

**Reply by juliettepaul** [reply]
Ah yep. I needed a computer restart and your new command works. I restarted sketchybar and now I have app icons. Thanks for making this bar, it is really awesome
